cabot/metricsapp/forms/grafana_elastic.py

```diff
@@ -68,3 +68,10 @@ def __init__(self, *args, **kwargs):
         self.fields['queries'].widget = forms.Textarea(attrs=dict(readonly='readonly',
                                                                   style='width:100%'))
         self.fields['queries'].help_text = None
+
+    def save(self):
+        """Adjust extended_bounds part of queries if the time_range is changed"""
+        model = super(GrafanaElasticsearchStatusCheckUpdateForm, self).save()
+        model.queries = json.dumps(adjust_time_range(json.loads(model.queries), model.time_range))
```

Comment:
Isn't there a better place to do this? IIRC, Django had some hooks for cleaning up each field before saving. Can we use that.